tests/checkpointing/test_model_checkpoint.py

```diff
@@ -1194,8 +1194,8 @@ def make_assertions(cb_restore, written_ckpt):
             "kth_best_model_path": False,
             "kth_value": False,
             "best_k_models": False,
+            "last_model_path": False,
```

Comment:
do we need the other additional test? wouldn't this one satisfy the requirement for the requirement we need?

Reply:
This test is more contained and just checks that the state is not restored.

But we could have a bug in the "save last" saving logic after reload which should be covered by the new one.

So you are correct depending on how lenient we want to be with our tests.